src/utils.ts

```diff
@@ -98,6 +98,9 @@ export function formatSeedId(id: string): string {
 }
 
 export function formatRadicleUrn(id: string): string {
+  // If the passed id has no prefix, add the default prefix
+  if (! id.startsWith("rad:git:")) id = "rad:git:" + id;
```

Comment:
Maybe better to assert that it *does* start, otherwise the output will be strange anyway.

Reply:
Or just `assert(isRadicleUrn(id))` if we have a function like that.